### PR TITLE
Publishing to pub.dev

### PR DESCRIPTION
**Pull request by dcupidon**
We have completed the changes needed for the package to be published on pub.dev.

### CODE REVIEW

**Review by sgong-pdftron — APPROVED** [review]

File: doc/api/API.md [hunk 1]
```diff
@@ -52,20 +52,20 @@ This section is for viewer related non-static methods. They would be callable in
...
 Widget (DocumentViewController):
 ```dart
-void _onDocumentViewCreated(DocumentViewController controller) {
-    controller.openDocument('https://pdftron.s3.amazonaws.com/downloads/pl/PDFTRON_about.pdf');
+void _onDocumentViewCreated(DocumentViewController controller) async {
+  await controller.openDocument('https://pdftron.s3.amazonaws.com/downloads/pl/PDFTRON_about.pdf');
 }
 ```
 
 We suggest that you stick with either version for the APIs that are callable in both versions, to avoid unnecessary problems.
```
Comment (JamesTron) [inline doc/api/API.md:66]: I don't quite understand this sentence. Do you mean choose either widget or plugin, but not both?

File: lib/constants.dart [hunk 2]
```diff
@@ -161,23 +161,25 @@ class Tools {
   static const formCreateComboBoxField = 'FormCreateComboBoxField';
   static const formCreateListBoxField = 'FormCreateListBoxField';
 
-  // iOS only.
+  /// iOS only.
   static const pencilKitDrawing = 'PencilKitDrawing';
 
-  // Android only.
+  /// Android only.
   static const annotationSmartPen = 'AnnotationSmartPen';
```
Comment (JamesTron) [inline lib/constants.dart:168]: This exists in iOS native too.

**Review by JamesTron — APPROVED** [review]
See comments. Would be nice to clarify first one, second one can be added to a backlog somewhere.

File: doc/api/API.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ void _onDocumentViewCreated(DocumentViewController controller) async {
 }
 ```
 
-We suggest that you stick with either version for the APIs that are callable in both versions, to avoid unnecessary problems.
+We suggest that you pick either widget or plugin and stick with that choice for all APIs, excluding APIs that are only available in one version.
```
Comment (JamesTron) [inline doc/api/API.md:66]:
```suggestion
You must choose either the widget or plugin, and use it for all APIs. Mixing widget and plugin APIs will not function correctly.
```

**Review by JamesTron — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# 1.0.0-beta.5 - Februrary 22, 2021
+
+- Toggle widget and plugin
```
Comment (sgong-pdftron) [inline CHANGELOG.md:3]: this is done for internal sample so not actually related to customer, should either remove, or mention it's only for the internal sample

File: README.md [hunk 5]
```diff
@@ -23,26 +23,29 @@ More information can be found at https://www.pdftron.com/documentation/guides/fl
...
 - No license key is required for trial. However, a valid commercial license key is required after trial.
 - PDFTron SDK >= 6.9.0
-- Flutter >= 1.0.0
+- Flutter >= 2.0.0
 
 ## Null Safety
-Dart now supports [sound null safety](https://dart.dev/null-safety), which is available starting from Dart 2.12.0 and Flutter 2.0.0. 
-
-Upgrading may cause breaking changes, so if you are not ready to change versions, continue using PDFTron Flutter SDK as you have been. 
+Dart now supports [sound null safety](https://dart.dev/null-safety), which is available starting from Dart 2.12.0 and Flutter 2.0.0.  Upgrading may cause breaking changes, so if you are not ready to change versions, refer to our [master branch](https://github.com/PDFTron/pdftron-flutter).
```
Comment (sgong-pdftron) [inline README.md:41]: this needs to be updated because master is also null safety

File: README.md [hunk 6]
```diff
@@ -23,26 +23,29 @@ More information can be found at https://www.pdftron.com/documentation/guides/fl
...
 
 If you would like to use our null safe SDK, it is available in the following places:
-- [GitHub](https://github.com/PDFTron/pdftron-flutter/tree/publish-prep)
+- [GitHub](https://github.com/PDFTron/pdftron-flutter/tree/publish-prep) (You are currently on this branch)
```
Comment (sgong-pdftron) [inline README.md:44]: missing update

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -23,26 +23,29 @@ More information can be found at https://www.pdftron.com/documentation/guides/fl
...
 - No license key is required for trial. However, a valid commercial license key is required after trial.
 - PDFTron SDK >= 6.9.0
-- Flutter >= 1.0.0
+- Flutter >= 2.0.0
 
 ## Null Safety
-Dart now supports [sound null safety](https://dart.dev/null-safety), which is available starting from Dart 2.12.0 and Flutter 2.0.0. 
-
-Upgrading may cause breaking changes, so if you are not ready to change versions, continue using PDFTron Flutter SDK as you have been. 
+Dart now supports [sound null safety](https://dart.dev/null-safety), which is available starting from Dart 2.12.0 and Flutter 2.0.0.
```
Comment (sgong-pdftron) [inline README.md:41]: should link to some official migration guide in case client is not ready for it

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

**Review by sgong-pdftron — APPROVED** [review]

**Pull request merged**